docker/Dockerfile

```diff
@@ -16,16 +16,21 @@ FROM openzipkin/zipkin-builder
 
 WORKDIR /code
 COPY . /code
-RUN mvn -B --no-transfer-progress package -DskipTests=true -pl autoconfigure/storage-stackdriver -am
+RUN mvn -B --no-transfer-progress package -DskipTests=true -pl module/storage-stackdriver -am
 
-RUN \
-  cd /code && mkdir -p /zipkin-gcp/stackdriver && cp autoconfigure/storage-stackdriver/target/zipkin-autoconfigure-storage-stackdriver-*-module.jar /zipkin-gcp/stackdriver && cd /zipkin-gcp/stackdriver && jar xf *.jar && rm *.jar
+WORKDIR /zipkin-gcp
+
+RUN cp /code/module/storage-stackdriver/target/zipkin-module-storage-stackdriver-*-module.jar stackdriver.jar && \
+  echo > .stackdriver_profile && \
+  (mkdir stackdriver && cd stackdriver && jar -xf ../stackdriver.jar) && \
+  rm stackdriver.jar
 
 FROM openzipkin/zipkin:master
 MAINTAINER Zipkin "https://zipkin.io/"
 
-COPY --from=0 /zipkin-gcp/ /zipkin/lib/
+COPY --from=0 /zipkin-gcp/ /zipkin/
 
-RUN echo > .xray_profile
+# Readback is currently not supported
+ENV QUERY_ENABLED false
 
-ENV MODULE_OPTS="-Dloader.path=lib/stackdriver -Dspring.profiles.active=stackdriver"
+env MODULE_OPTS="-Dloader.path=stackdriver -Dspring.profiles.active=stackdriver"
```

Comment:
`env` should still be in caps like our other dockerfile commands

Reply:
https://github.com/openzipkin/zipkin-gcp/commit/aa53ef4be478be51c60de3bc335ad074d760a6aa thx it was copy/pasta from the other image which I'll also fix